src/rgw/rgw_main.cc

```diff
@@ -401,6 +402,11 @@ int radosgw_Main(int argc, const char **argv)
     if (!rgw::kafka::init(cct.get())) {
         dout(1) << "ERROR: failed to initialize Kafka manager" << dendl;
     }
+#endif
+#ifdef WITH_RADOSGW_AWS_ENDPOINT
+  if (!rgw::aws::init(cct.get())) {
```

Comment:
we're adding global init/shutdown to rgw_main.cc - do we need it in rgw_admin.cc or librgw.cc?

Reply:
not needed in rgw_admin
*is* needed in librgw (to avoid the persistent notification "ghost rgw" bug...) - will fix!